BitTornado/Tracker/track.py

```diff
@@ -703,10 +703,10 @@ def params(key, default=None, l=paramslist):
             raise ValueError('invalid amount left')
         #uploaded = long(params('uploaded',''))
         #downloaded = long(params('downloaded',''))
-        if params('supportcrypto'):
+        if params('supportcrypto', 0):
```

Comment:
There's no need to add the 0 parameter, unless the goal is simply to provide a clue that the type of `params('supportcrypto')` should be numeric. The default `None` that `params` returns on failure will also evaluate `False`.

I would probably prefer a pull request that doesn't include this addition, as it's more of a style decision than a bug fix, and I'd want to be consistent about style changes.